setup.py

```diff
@@ -27,7 +27,8 @@
         'image_editor.core.add_elements',
         'image_editor.core.image_processing',
         'image_editor.gui',
-        'image_editor.utils'
+        'image_editor.utils',
+        'image_editor.test',
```

Comment:
remove this "image_editor.test" from packages list. Since we don't want user to access our unittest cases file, unittest is just for internal purpose. We will provide Sphinx documentation for usage.